lib/zendesk_api/client.rb

```diff
@@ -37,10 +37,11 @@ class Client
     def method_missing(method, *args, &block)
       method = method.to_s
       options = args.last.is_a?(Hash) ? args.pop : {}
-
+      resource_class = method_as_class(method)
```

Comment:
Great naming 💯

Reply:
makes me think that `method_as_class` could be renamed to `resource_class_for(method)`